### ISSUE TITLE
[serval] fan cleaning

### ISSUE THREAD

**Issue by 130s** [130s/hut_10sqft#149]
I can't remember when was the last time I cleaned the 2 fans at the bottom of Serval (purchased in 2011), but this time both were clogged like this. The room the computer sits in there's also a litter box for 2 cats. There was almost a month where 2 cats were kept in this room the entire period (https://github.com/130s/30y-130s_life/issues/166). And I saw the temperature on `Psensor` Ubuntu app some devices went to upper 80F degree, which is bad (I remember when CPU reached 90F the computer might automatically turned off).
![20170706_163310](https://user-images.githubusercontent.com/1840401/27967227-15035266-62f7-11e7-8097-78edf1bf34d9.jpg)

So as long as it stays in this catsy room, maybe once in an year at minimum to clean up.

**Comment by 130s** [comment]
Calendar event is set. No todos to be discussed here for now so closing.

**Comment by 130s** [comment]
Today while I was working on serval (for the first time since #282 occurred 1.5 months ago), while `apt-get dist-upgrade` was running the power turned off suddenly. I turned it on and found it was in a sleep/suspension. After a while it turned off by itself again.

Not sure if it's related, but I found the fan was clogged with dirt and mostly cats hair. So I removed them only from outside. Didn't open the lid this time.
